static/analysis.go

```diff
@@ -126,6 +154,23 @@ func analyzeHandler(
 					message.EventRole,
 				)
 			}
+		case "SchedulesTimeoutType":
+			if mi, ok := args[0].(*ssa.MakeInterface); ok {
+				hdr.MessageNamesValue.Produced.Add(
+					message.NameFromType(
+						mi.X.Type(),
+					),
+					message.TimeoutRole,
+				)
+			}
+			if mi, ok := args[0].(*ssa.MakeInterface); ok {
```

Comment:
```suggestion
```

Same `if` condition as above.

Reply:
Done in https://github.com/dogmatiq/configkit/pull/130/commits/71726203dc06831aeeea7c6d3f5542f5246fa8a8.